### ISSUE TITLE
au.csiro.aehrc.third.hail-is issue

### ISSUE THREAD

**Issue by ChristinaXu2017** [aehrc/VariantSpark#239]
## Problem
We follow the document to run `mvn clean install` but got below message:

```
[ERROR] Failed to execute goal on project variant-spark_2.12: Could not resolve dependencies for project au.csiro.aehrc.variant-spark:variant-spark_2.12:jar:0.5.3-SNAPSHOT: The following artifacts could not be resolved: au.csiro.aehrc.third.hail-is:hail_2.12_3.1:jar:all:0.2.74-SNAPSHOT (absent): Could not find artifact au.csiro.aehrc.third.hail-is:hail_2.12_3.1:jar:all:0.2.74-20210902.035537-2 in ossrh (https://oss.sonatype.org/content/repositories/snapshots)
```

## Investigation
I checked the mvn repo https://oss.sonatype.org/#nexus-search;quick~au.csiro.aehrc.third.hail-is; but the hail_2.12_3.1-0.2.74-SNAPSHOT-all.jar is not found in the repository.  While hail_2.12_3.1-0.2.74-SNAPSHOT.jar exists. 

I guess this two jar files hail_2.12_3.1-0.2.74-SNAPSHOT*.jar are same. Once we downloaded the one listed in the provided scope, it can be used for both run time and unit tests time.  Let's try to remove below block of dependency from the pom.xml
```
    <dependency>
      <groupId>au.csiro.aehrc.third.hail-is</groupId>
      <artifactId>hail_${scala.binary.version}_${spark.binary.version}</artifactId>
      <version>${hail.version}-SNAPSHOT</version>
      <classifier>all</classifier>
      <scope>test</scope>
    </dependency>
```

## solution  
- delete your local maven cache, eg. Mac maven local repo is /Users/<user>/.m2/repository
- update the pom.xml
- run `mvn clean install`  (it works on my Mac)

**Comment by NickEdwards7502** [comment]
Also resolves issue on Ubuntu

**Comment by ChristinaXu2017** [comment]
problem solved during pull request #240
